test/e2e/policies.test-suite/balloons/n4c16/test22-isolcpus/code.var.sh

```diff
@@ -0,0 +1,49 @@
+helm-terminate
+helm_config=${TEST_DIR}/balloons-isolcpus.cfg helm-launch balloons
+
+vm-command "grep isolcpus=0,1 /proc/cmdline" || {
+    if [[ "$distro" == *fedora* ]]; then
+        fedora-set-kernel-cmdline "isolcpus=0,1"
+    else
+        ubuntu-set-kernel-cmdline "isolcpus=0,1"
+    fi
+    vm-reboot
+    sleep 5
+    vm-command "grep isolcpus=0,1 /proc/cmdline" || {
+        error "failed to set isolcpus kernel commandline parameter"
+    }
+    vm-command "systemctl restart kubelet"
+    sleep 5
+    vm-wait-process --timeout 120 kube-apiserver
```

Comment:
This is not enough. After a node reboot, none of the control plane containers are running. Waiting for the API server is not enough. It will take a long time after it is running before the node is really fully bootstrapped again.
Since ATM we always use cilium as our CNI, you could use something like this:

```
restart-kubelet() {
    vm-command "systemctl restart kubelet"
    sleep 5
    vm-wait-process --timeout 120 kube-apiserver
    vm-command "cilium status --wait --interactive=false"
}
```
and then call `restart-kubelet` to restart the kubelet and wait for the node to become service ready.